src/components/Nav/Styled.Nav.js

```diff
@@ -0,0 +1,95 @@
+import styled from 'styled-components';
+
+const S = {};
+
+S.Header = styled.header`
```

Comment:
```suggestion

export const Header = styled.header`
```
- 아래에도 모두 이렇게 export를 하시고 Nav에서 아래와 같이 import 하시면 새로운 S라는 객체를 만들지 않고도 할 수 있는 방법이 있습니다.
```
import * as S from './Styled.Nav'; 
```
- 방법에 대해서만 말씀드리는거라 굳이 고치지 않으셔도 됩니다. 
- 하지만 심심하시면 한번 해보시는 것도 추천드립니다.